Makefile

```diff
@@ -83,7 +83,7 @@ functest-image-push: functest-image-build
 	hack/func-tests-image.sh push
 
 conformance:
-	hack/dockerized "hack/conformance.sh"
+	hack/dockerized "export SKIP_OUTSIDE_CONN_TESTS=${SKIP_OUTSIDE_CONN_TESTS} hack/conformance.sh"
```

Comment:
@alonSadan That line didn't work for me. I had to fix this with `&&` for example: `export SKIP_OUTSIDE_CONN_TESTS=${SKIP_OUTSIDE_CONN_TESTS} && hack/conformance.sh`

Reply:
Done.